src/components/ContentWidgets/ToDoList/ToDoListEditor.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import { ToDoListItem } from './ToDoList';
+
+export const ToDoListEditor: React.FC<WidgetEditorProps> = ({
+    originalContent,
+    editedContent,
+    setEditedContentOnChange
+}) => {
+    let listContent = null;
+
+    // let findItemIndex = (findItemID: number, copyEditedContent: ToDoListItem[]) => {
+        
+    //     for (let i = 0; i < copyEditedContent.length; i++) {
+    //         if (findItemID == copyEditedContent[i].toDoListItem_order) {
+    //             return i;
+    //         }
+    //     }
+    //     return -1;
+    // }
+
+    if (editedContent.toDoList_content != null) {
+        listContent = editedContent.toDoList_content.map((item) => {
+            
+            let copyEditedContent = [...editedContent.toDoList_content!!];
+
+            let removeAction = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+                let index = parseInt(e.currentTarget.parentElement!!.id);
+                copyEditedContent.splice(index, 1);
+                // reassigns ordering
```

Comment:
You shouldn't have to reassign ordering if you're trying to do it a similar way to how I did the widget loading. You would splice the order array and update that, as well as delete the key from your todo object.